README.md

```diff
@@ -21,14 +21,44 @@ For DevOps or SRE teams this charm will make operating WordPress simple and stra
 It will allow easy deployment into multiple environments for testing of changes, 
 and supports scaling out for enterprise deployments.
 
+For information about how to deploy, integrate, and manage this charm, see the Official [wordpress-k8s-operator Documentation](https://charmhub.io/wordpress-k8s/docs).
+
+## Get Started
+
+To begin, refer to the [Getting Started](https://charmhub.io/wordpress-k8s/docs/tutorial) tutorial for step-by-step instructions.
+
+### Basic Operations
+
+The following actions are available for the charm - 
+- get-initial-password
+- rotate-wordpress-secrets
+
+You can find more information on this [here](https://charmhub.io/wordpress-k8s/actions)
+
+## Integrations
+
+Deployment of WordPress requires a relational database. The integration with the mysql interface is required by the wordpress-k8s charm for which `mysql-k8s` charm can be deployed as follows -
```

Comment:
Nitpick: Use a colon at the end instead of a dash

Reply:
addressed